<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>945</width>
    <height>405</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Font Size Type</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Family</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>270</y>
      <width>771</width>
      <height>20</height>
     </rect>
    </property>
    <widget class="QRadioButton" name="ArialChk">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Arial</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="TimesNewRomanChk">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>0</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Times New Roman</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="ComicSansMSChk">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>0</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Comic Sans MS</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>190</y>
      <width>521</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>300</y>
      <width>521</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Font Size</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Font Size Type</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>230</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>small</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>medium</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>big</string>
     </property>
    </item>
   </widget>
   <widget class="TargetLabel" name="ChngLabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>701</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>The quick brown fox likes CSE 335.</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>945</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TargetLabel</class>
   <extends>QLabel</extends>
   <header location="global">targetlabel.h</header>
   <slots>
    <slot>setArial()</slot>
    <slot>setTimesNewRoman()</slot>
    <slot>setComicSansMS()</slot>
    <slot>setFontSize(int)</slot>
    <slot>setSizeByFamily(QSting)</slot>
    <slot>setSizeByFamilyText(QString)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>comboBox</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>lineEdit</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>671</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>665</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>comboBox</receiver>
   <slot>setCurrentText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>346</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>560</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
